packages/grafana-data/src/transformations/transformers/calculateField.test.ts

```diff
@@ -4,9 +4,10 @@ import { FieldType } from '../../types/dataFrame';
 import { ReducerID } from '../fieldReducer';
 import { mockTransformationsRegistry } from '../../utils/tests/mockTransformationsRegistry';
 import { transformDataFrame } from '../transformDataFrame';
-import { calculateFieldTransformer, CalculateFieldMode, ReduceOptions } from './calculateField';
+import { CalculateFieldMode, calculateFieldTransformer, ReduceOptions } from './calculateField';
 import { DataFrameView } from '../../dataframe';
 import { BinaryOperationID } from '../../utils';
+import { observableTester } from '../../../../../public/test/helpers/observableTester';
```

Comment:
Usually, this kind of import in package gives me gooseflesh 😬 Wonder if we shouldn't move test helpers to separate package (grafana/test-utils ?) to make sure we don't break anything at some point. Just an idea, definitely not in the scope of this PR :)

Reply:
Hmm, you're right that isn't good at all. Strange that the linting doesn't prevent me from this?

Reply:
Have you synced with latest master? Because with the updates med and Arve did last week the PR build should fail when this happens.

Reply:
I synced with master this morning and the build passed?

Reply:
Moved observableTester to grafana/data instead until we add the testutils packages